lib/api/models/request.js

```diff
@@ -21,7 +19,7 @@ export class Request {
   buildMockedRequestData() {
 
     const defaultState = {
-      name: 'Original Response',
+      name: 'Mocked Response',
```

Comment:
All responses are mocked responses. An original response is the original response from the server.
It should be kept as original response.